package.json

```diff
@@ -1,18 +1,11 @@
 {
-  "name": "ng-table-virtual-scroll",
-  "version": "1.3.6",
+  "name": "@craftworks/ng-table-virtual-scroll",
+  "version": "1.3.7",
   "description": "Virtual scroll for for Angular Material Table",
```

Comment:
maybe we can add the link to the original homepage here in the description?

```suggestion
  "description": "Virtual scroll for for Angular Material Table, fork of https://github.com/diprokon/ng-table-virtual-scroll",
```

Reply:
like how else? this the link to the original repo?

Reply:
isn't this the original repo link. i just thought maybe its a good idea to add this link to the description. but not sure :)